modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/components/containers/Forms/Basics/BusinessType/RadioGroup.js

```diff
@@ -30,6 +30,10 @@ export const BusinessTypeRadioGroup = ({businessTypes = []}) => {
 				'basics.properties.businessClassCode',
 				categoryProperties.find(({key}) => key === 'BCC').value
 			);
+			setValue(
+				'basics.properties.naics',
+				categoryProperties.find(({key}) => key === 'NAICS').value
```

Comment:
@jairmedeiros , I think we should use ?.value (Optional Chaining) for those finds, because some business doesn't have a NAICS

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/22279592/129618760-ddbbb2b4-c2a3-4b57-8628-9a189712cf33.png">

See this Image

Reply:
done!